dotnet/src/Agents/Abstractions/Extensions/ChatMessageContentExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.SemanticKernel.Agents.Extensions;
+
+/// <summary>
+/// Extension methods for <see cref="ChatMessageContent"/>
+/// </summary>
+internal static class ChatMessageContentExtensions
+{
+    /// <summary>
+    /// Determines if <see cref="ChatMessageContent"/> has content.
+    /// </summary>
+    public static bool HasContent(this ChatMessageContent? message)
+        => !string.IsNullOrWhiteSpace(message?.Content);
```

Comment:
This logic will check only string/text content and does not take into account the other types of content like audio, image, etc.